kuyruk_sentry.py

```diff
@@ -27,18 +24,15 @@ def capture_exception(self,
                           worker=None,
                           queue=None,
                           **extra):
-        with self.hub.push_scope() as scope:
-            extras = {
-                        "description": description,
-                        "queue": queue,
-                        "worker_hostname": socket.gethostname(),
-                        "worker_pid": os.getpid(),
-                        "worker_cmd": ' '.join(sys.argv),
-                        "worker_timestamp": datetime.utcnow().isoformat()[:19],
-                    }
-            for key, value in extras.items():
-                scope.set_extra(key, value)
-            sentry_id = self.hub.capture_exception(exc_info)
```

Comment:
Hub-based APIs and `push_scope` is [deprecated](https://docs.sentry.io/platforms/python/migration/1.x-to-2.x#custom-instrumentation). Seems like we only do this to set extras. `capture_exception(..., extras=extras)` does the same. Tested with 1.45.0 & 2.7.0, but 0.15.1 (min supported) still looks the same looking at code.